docs/3x/PyTorch.md

```diff
@@ -0,0 +1,230 @@
+Torch
+=================================================
+
+1. [Introduction](#introduction)
+2. [Torch-like APIs](#torch-like-apis)
+3. [Support matrix](#supported-matrix)
+4. [Common Problems](#common-problems)
+
+## Introduction
+
+`neural_compressor.torch` provides a Torch-like API and integrates various model compression methods fine-grained to the torch.nn.Module. Supports a comprehensive range of models, including but not limited to CV models, NLP models, and large language models. A variety of quantization methods are available, including classic INT8 quantization, SmoothQuant, and the popular weight-only quantization. Neural compressor also provides the latest research in simulation work, such as FP8 emulation quantization, MX data type emulation quantization.
+
+In terms of ease of use, neural compressor is committed to providing an easy-to-use user interface and easy to extend the structure design, on the one hand, reuse the PyTorch prepare, convert API, on the other hand, through the Quantizer base class for prepare and convert customization to provide a convenient.
+
+For more details, please refer to [link](https://github.com/intel/neural-compressor/discussions/1527) in Neural Compressor discussion space.
+
+So far, `neural_compressor.torch` still relies on the backend to generate the quantized model and run it on the corresponding backend, but in the future, neural_compressor is committed to provide generalized device-agnostic Q-DQ model, so as to realize one-time quantization and arbitrary deployment.
```

Comment:
how about changing "comitted" to "planned"? which leave us some room in case of feasibility issue in certain backend.  it is better replace "realize" to "achieve"